graspologic/inference/latent_position_test.py

```diff
@@ -1,20 +1,22 @@
 # Copyright (c) Microsoft Corporation and contributors.
 # Licensed under the MIT License.
-
+# Test
 import numpy as np
+
 from scipy.linalg import orthogonal_procrustes
 
 from ..align import OrthogonalProcrustes
 from ..embed import AdjacencySpectralEmbed, OmnibusEmbed, select_dimension
 from ..simulations import rdpg
 from ..utils import import_graph, is_symmetric
-from .base import BaseInference
 
 
-class LatentPositionTest(BaseInference):
+def latent_position_test(
+    A1, A2, embedding="ase", n_components=None, test_case="rotation", n_bootstraps=500
+):
     r"""
-    Two-sample hypothesis test for the problem of determining whether two random
-    dot product graphs have the same latent positions.
+    Two sample hypothesis test for the problem of determining whether two random dot
```

Comment:
```suggestion
    Two-sample hypothesis test for the problem of determining whether two random dot
```
- should be there. revert if changed anywhere else, please.